90_mender_boot_grub.cfg

```diff
@@ -10,7 +10,17 @@ if test -n "${kernel_devicetree}"; then
     fi
 fi
 
-if linux (${mender_grub_storage_device},${ptable_type}${mender_boot_part})/boot/${kernel_imagetype} root=${mender_kernel_root_base}${mender_boot_part} ${bootargs}; then
+if [ test -n "${mender_rootfsa_uuid}" -a test -n  "${mender_rootfsb_uuid}" ]; then
+    if [ "${mender_boot_part}" = "${mender_rootfsa_part}" ]; then
+        mender_root="PARTUUID=${mender_rootfsa_uuid}"
+    else
+        mender_root="PARTUUID=${mender_rootfsb_uuid}"
+    fi
```

Comment:
I think I would change the `else` into `elif [ "${mender_boot_part}" = "${mender_rootfsb_part}" ]; then`, and leave `mender_root` unassigned if `mender_boot_part` doesn't match either. Pretty unlikely, but if it does happen, and it is still able to load a kernel, I would rather have the error pop up immediately because the root is not found, rather than silently mounting the wrong filesystem.